docs/storage/source/storage_hdf5.rst

```diff
@@ -102,14 +103,15 @@ Datasets
     linkable                      Not mapped; Stored in schema only
```

Comment:
I don't think ``linkable`` is part of the schema language any more, I think this is left-over from NWB 1.

Reply:
It looks like it is still in the schema language. Just not used. https://schema-language.readthedocs.io/en/latest/specification_language_description.html?highlight=linkable#groups